README.md

```diff
@@ -3112,11 +3122,11 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
-<a name="ecmascript-6-styles"></a>
+<a name="ecmascript-6+-styles"></a>
```

Comment:
this will cause previous links to be broken - it needs to stay the same forever. if you'd like, we can add a second link, but i'm not sure it adds much.